.travis.yml

```
@@ -8,7 +8,8 @@ jobs:
     python: 2.7
   - <<: *latest_py2
     env: LANG=C
-  - python: pypy
+  - python: pypy2.7
```

You should set a specific version, otherwise you might actually get something older.
```suggestion
  - python: pypy2.7-6.0.0
```